index.html

```diff
@@ -0,0 +1,288 @@
+          onclick="partyCloseHamburger()"
+          id="partyModeHamburger"
+          class="partyButton"
+        >Party mode</button></span>
+    </ul>
+  </nav>
+
+  <!---HERO SECTION-->
+  <header
+    id="homePage"
+    class="hero"
+  >
+    <div class="header-text">
+      <h1 id="heroText">WILLZ MODELS</h1>
+      <p class="header-paragraph">A one cat model agency</p>
+    </div>
+  </header>
+
+  <!----MAIN section--------->
+  <section
+    id="modelSection"
+    class="main-section"
+  >
+    <div class="card 1">
+      <img src="images/willz-zara.jpeg" />
+      <h3>Fashionable</h3>
+    </div>
+
+    <div class="card 2">
+      <img src="images/Willz-fancy.jpeg" />
+      <h3>Sofisticated</h3>
+    </div>
+
+    <div class="card 3">
+      <img src="images/willz-cowboy.jpeg" />
+      <h3>casual</h3>
+    </div>
+
+    <div class="card 4">
+      <img src="images/Willz-catwalk.jpeg" />
+      <h3>professional</h3>
+    </div>
+
+    <div class="card 5">
+      <img src="images/willz-royal.jpeg" />
+      <h3>Royal</h3>
+    </div>
+
+    <div class="card 6">
+      <img src="images/willz-potato.jpeg" />
+      <h3>stoic</h3>
+    </div>
+
+    <div class="card 7">
+      <img src="images/willz-kpop.jpeg" />
+      <h3>Modern</h3>
+    </div>
+
+    <div class="card 8">
+      <img src="images/willz-jacket.jpeg" />
+      <h3>Focused</h3>
+    </div>
+
+    <div class="card 9">
+      <img src="images/willz-princess.jpeg" />
+      <h3>Feminine</h3>
+    </div>
+
+    <div class="card 10">
+      <img src="images/willz-artsy.jpeg" />
+      <h3>Artistic</h3>
+    </div>
+
+    <div class="card 11">
+      <img src="images/willz-manicure.jpeg" />
+      <h3>Well groomed</h3>
+    </div>
+
+    <div class="card 12">
+      <img src="images/willz-nature.jpeg" />
+      <h3>Natural</h3>
+    </div>
+  </section>
+
+
+  <!--INFO + FORM section------->
+  <section
+    id="information"
+    class="info-section"
+  >
+    <div class="info-text">
+      <h2>ABOUT US</h2>
+      <img
+        class="infoImage"
+        src="images/willz-dreaming.png"
+        alt="cat"
+      >
+      <Professional>Forget about casting calls, multiple models, or diva drama. Willz is the one-cat wonder your brand
+        has been dreaming of. From slick fashion to casual cool, Willz can pull off any look, any pose and any vibe. All
+        without a single hiss. Professional Purr-fectionist: Takes direction better than most humans. Multi-Look expert:
+        From well dressed sofisticated looks to natural vibes, Willz does it all. Flexible & Focused: Stretches, yawns,
+        or stares
+        dramatically, all on cue. Why hire a dozen models when Willz has mastered it all? Book a shoot and let this
+        feline phenomenon turn you project into a paw-some masterpiece.
+    </div>
+
+    <div
+      id="formSection"
+      class="form-section"
+    >
+      <form
+        action="http://httpbin.org/anything"
```

Comment:
Make sure to use the https method, as some browser will otherwise block this action